### DISCUSSION TITLE
Allowed Block Types

### DISCUSSION THREAD

**Discussion by bdfpplinsinga** [cearls/timberland#37] [Q&A] [answered]
Hee Chris,

Is there a reason you removed the allowed_block_types_all filter in your new vite version of the theme? Are you using core blocks these days? I still think its quite the unexpected UI when mixing core blocks with ACF blocks.

**Comment by cearls** [comment, marked as answer]
Hey @bdfpplinsinga! I found that I wasn't using it in projects and custom blocks are combined with core blocks from time to time. So, I removed it to clean up the functions file. It was commented out in the previous version and I think it would be fine to add it back as a reference in case someone wants to use it.

**Reply by bdfpplinsinga** [reply]
Hey Chris! Thanks for the message. I'm really interested in seeing how you use custom blocks in combination with core blocks. I've come across a few projects that attempted this, but haven't seen any standout examples yet. I'd love to see some screens or code examples if you have any.

**Comment by bdfpplinsinga** [comment]
One more thing. Currious as i am. I wanted to test some innerblocks again. But i noticed that when i set the jsx to true on the example block, this breaks the block. Any idea why? Is this working with you new setup? I tested in you older one with laravel, and here it is still working.

**Reply by bdfpplinsinga** [reply]
I get a json error when i set jsx to true. 

```javascript
react-dom.min.js?ver=18.2.0:10 SyntaxError: Expected property name or '}' in JSON at position 2 (line 1 column 3)
```

**Reply by cearls** [reply]
React doesn't like the Alpine syntax (e.g. `x-data="{}"`). I guess I've gotten away with it because I've only used jsx and `<InnerBlocks/>` on custom blocks that don't use Alpine like `column`. It will probably take a workaround like setting a twig variable based on `$is_preview`.

**Reply by cearls** [reply]
Actually, I've gotten around this in the past by moving `x-data` into `Alpine.data`.

[https://alpinejs.dev/globals/alpine-data](https://alpinejs.dev/globals/alpine-data)

**Reply by bdfpplinsinga** [reply]
Thanx man! For getting back on this.
